app/styles/ember-appuniversum/_c-toaster.scss

```diff
@@ -0,0 +1,19 @@
+.au-c-toaster {
+  position: fixed;
+  display: flex;
+  flex-direction: column-reverse;
+  gap: $au-unit-small;
+  bottom: 0;
```

Comment:
Added the option to position on the top. I guess it's a preference and Kaleidos already uses the bottom alignment.

Reply:
Hmm, if it's just a preference thing I would just let Appuniversum make the decision? I think we want this consistent across all projects, no?

I only brought it up because I assumed the notification would be easier to notice in the top right corner, but if that's not the case I'm fine with keeping the bottom positioned version.

Reply:
Not sure, Kaleidos and ABB applications have quite a different approach and legacy currently. I'm sure we could move toward something consistent in the future. In ABB apps moving it to the top would probably be more visible. The option is there to support both approaches for the moment. Maybe I should use the top position as default.